src/main/java/HumanPlayer.java

```diff
@@ -1,27 +1,35 @@
-import java.util.Scanner;
-
 public class HumanPlayer implements Player {
     private Display display;
     private String mark;
-    private Scanner scanner;
-
+    private InputValidator inputValidator;
 
     public HumanPlayer
-            (Display display, String mark, Scanner scanner) {
+            (Display display, String mark, InputValidator inputValidator) {
         this.display = display;
         this.mark = mark;
-        this.scanner = scanner;
-
+        this.inputValidator = inputValidator;
     }
 
     public String getMark() {
         return mark;
     }
 
     @Override
-    public int choose_move(Board board) {
-        display.make_move();
-        var move = scanner.nextInt();
+    public int chooseMove(Board board, String opponentMark) {
+        display.showGrid(board.getGrid());
+        var move = 0;
```

Comment:
Is this necessary?

Reply:
@Kotauror Which part are you referring to? Displaying the grid?

Reply:
@itsellej - `var move = 0`